mmv1/products/dlp/JobTrigger.yaml

```diff
@@ -158,6 +175,413 @@ properties:
+                  name: 'exclusionType'
+                  description: |
+                    If set to EXCLUSION_TYPE_EXCLUDE this infoType will not cause a finding to be returned. It still can be used for rules matching.
+                  values:
+                    - :EXCLUSION_TYPE_EXCLUDE
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'regex'
+                  description: Regular expression which defines the rule.
+                  immutable: true
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: 'pattern'
+                      required: true
+                      description: |
+                        Pattern defining the regular expression.
+                        Its syntax (https://github.com/google/re2/wiki/Syntax) can be found under the google/re2 repository on GitHub.
+                    - !ruby/object:Api::Type::Array
+                      name: 'groupIndexes'
+                      description: |
+                        The index of the submatch to extract as findings. When not specified, the entire match is returned. No more than 3 may be included.
+                      item_type: Api::Type::Integer
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'dictionary'
+                  description: Dictionary which defines the rule.
+                  immutable: true
+                  properties:
+                    - !ruby/object:Api::Type::NestedObject
+                      name: 'wordList'
+                      description: List of words or phrases to search for.
+                      properties:
+                        - !ruby/object:Api::Type::Array
+                          name: 'words'
+                          required: true
+                          description: |
+                            Words or phrases defining the dictionary. The dictionary must contain at least one
+                            phrase and every phrase must contain at least 2 characters that are letters or digits.
+                          item_type: Api::Type::String
+                    - !ruby/object:Api::Type::NestedObject
+                      name: 'cloudStoragePath'
+                      description: Newline-delimited file of words in Cloud Storage. Only
+                        a single file is accepted.
+                      properties:
+                        - !ruby/object:Api::Type::String
+                          name: 'path'
+                          required: true
+                          description: |
+                            A url representing a file or path (no wildcards) in Cloud Storage. Example: `gs://[BUCKET_NAME]/dictionary.txt`
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'storedType'
+                  description: A reference to a StoredInfoType to use with scanning.
+                  immutable: true
```

Comment:
Is `immutable: true` is required here?

Reply:
I've checked this field and it runs fine without `ForceNew`. Hence, removing `immutable:true`.